contrib/infiniband/ibvidentifier.h

```diff
@@ -1,26 +1,22 @@
 #ifndef IBVID_H
 #define IBVID_H
-#include <sys/types.h>
-#include <infiniband/verbs.h>
-#include <limits.h>
-#include <unistd.h>
-#include <stdlib.h>
+#include <stdint.h>
 
-struct ibv_qp_id {
+typedef struct ibv_qp_id {
   uint32_t qpn;
   uint16_t lid;
   uint32_t psn;
-};
+} ibv_qp_id;
```

Comment:
Let's change the new `typedef` type from `ibv_qp_id` to `ibv_qp_id_t` everywhere.  This is closer to standard conventions.  (`_t` for "type")